docs/zh-cn/_summary.md

```diff
@@ -1,133 +1,192 @@
+    * [光标设备](zh-cn/feature_pointing_device.md)
+    * [Raw HID](zh-cn/feature_rawhid.md)
+    * [Sequencer](zh-cn/feature_sequencer.md)
+    * [换手](zh-cn/feature_swap_hands.md)
+    * [多击键](zh-cn/feature_tap_dance.md)
+    * [点按配置](zh-cn/tap_hold.md)
+    * [终端](zh-cn/feature_terminal.md)
+    * [Unicode](zh-cn/feature_unicode.md)
+    * [用户空间](zh-cn/feature_userspace.md)
+    * [WPM计算](zh-cn/feature_wpm.md)
+
+  * 硬件特性
+    * 显示
+      * [HD44780 LCD控制器](zh-cn/feature_hd44780.md)
+      * [ST7565 LCD驱动](zh-cn/feature_st7565.md)
+      * [OLED驱动](zh-cn/feature_oled_driver.md)
+    * 灯效
+      * [背光](zh-cn/feature_backlight.md)
+      * [LED矩阵](zh-cn/feature_led_matrix.md)
+      * [RGB灯光](zh-cn/feature_rgblight.md)
+      * [RGB矩阵](zh-cn/feature_rgb_matrix.md)
+    * [音频](zh-cn/feature_audio.md)
+    * [蓝牙](zh-cn/feature_bluetooth.md)
+    * [Bootmagic Lite](zh-cn/feature_bootmagic.md)
+    * [自定义矩阵](zh-cn/custom_matrix.md)
+    * [Digitizer](zh-cn/feature_digitizer.md)
+    * [拨动开关（DIP Switch）](zh-cn/feature_dip_switch.md)
+    * [编码器（滚轮）](zh-cn/feature_encoders.md)
+    * [触摸反馈](zh-cn/feature_haptic_feedback.md)
+    * [摇杆](zh-cn/feature_joystick.md)
+    * [LED指示](zh-cn/feature_led_indicators.md)
+    * [MIDI](zh-cn/feature_midi.md)
+    * [Proton C转换](zh-cn/proton_c_conversion.md)
+    * [PS/2鼠标](zh-cn/feature_ps2_mouse.md)
+    * [分体式键盘](zh-cn/feature_split_keyboard.md)
+    * [速记](zh-cn/feature_stenography.md)
+    * [热敏打印机](zh-cn/feature_thermal_printer.md)
+    * [Velocikey](zh-cn/feature_velocikey.md)
+
+* QMK开发
+  * [PR Checklist](zh-cn/pr_checklist.md)
+  * 打破兼容的改动
+    * [总览](zh-cn/breaking_changes.md)
+    * [我的PR已打上标记](zh-cn/breaking_changes_instructions.md)
+    * [近期的变更日志(Changelog)](zh-cn/ChangeLog/20210529.md "QMK v0.13.0 - 2021 May 29")
+    * [更早期的不兼容改动](zh-cn/breaking_changes_history.md)
+
+  * C语言开发
+    * [ARM调试指引](zh-cn/arm_debugging.md)
+    * [AVR处理器](zh-cn/hardware_avr.md)
+    * [C编码规范](zh-cn/coding_conventions_c.md)
+    * [兼容的微处理器](zh-cn/compatible_microcontrollers.md)
+    * [驱动](zh-cn/hardware_drivers.md)
+      * [ADC驱动](zh-cn/adc_driver.md)
+      * [Audio驱动](zh-cn/audio_driver.md)
+      * [I2C驱动](zh-cn/i2c_driver.md)
+      * [SPI驱动](zh-cn/spi_driver.md)
+      * [WS2812驱动](zh-cn/ws2812_driver.md)
+      * [EEPROM驱动](zh-cn/eeprom_driver.md)
+      * [串口驱动](zh-cn/serial_driver.md)
+      * [UART驱动](zh-cn/uart_driver.md)
+    * [操控GPIO](zh-cn/internals_gpio_control.md)
+    * [键盘开发指引](zh-cn/hardware_keyboard_guidelines.md)
+
+  * Python开发
+    * [编码规范](zh-cn/coding_conventions_python.md)
+    * [QMK CLI开发](zh-cn/cli_development.md)
+
+  * 配置器开发
+    * QMK API
+      * [开发环境](zh-cn/api_development_environment.md)
+      * [架构总览](zh-cn/api_development_overview.md)
+
+  * 硬件平台开发
+    * Arm/ChibiOS
+      * [选择MCU](zh-cn/platformdev_selecting_arm_mcu.md)
+      * [启动引导](zh-cn/platformdev_chibios_earlyinit.md)
+
+  * QMK参考信息
+    * [参与到QMK](zh-cn/contributing.md)
+    * [翻译QMK文档](translating.md)
```

Comment:
```suggestion
    * [翻译QMK文档](zh-cn/translating.md)
```

Reply:
I did not tend to translate this page, if anyone wants to do translating works, they should be fine with original english content.
and normal Chinese readers should not care that page.

Reply:
The idea is that it would fall back to translating.md, and when it's eventually translated it would be pointed correctly